package.json

```diff
@@ -35,6 +35,7 @@
     "postinstall": "precommit install",
     "coverage-test": "multi='xunit=shippable/testresults/result.xml' istanbul cover _mocha",
     "coverage-report": "istanbul report cobertura --dir shippable/codecoverage/",
+    "migrations": "NODE_ENV=productionAlter node utils/run-migrations.js",
```

Comment:
I might be mistaken, but here you refer to the node env as `productionAlter `, but the docker file  refers to it as `production`, is that correct?